app/client/components/dashboard/card/CustomTabs.jsx

```diff
@@ -15,7 +15,7 @@ import {
 const { TabPane } = Tabs;
 const { Text } = Typography;
 
-import { replaceUnderscores, toTitleCase } from "../../../utils";
+import { replaceUnderscores, toTitleCase, toDollars } from "../../../utils";
```

Comment:
I'm either going crazy or `toDollars` doesn't seem to be defined in `utils.js`.

Reply:
Also `toDollars` implies we will always be using "dollars"... should this be `toCurrency`?

Reply:
D'oh! Forgot to add that change to the PR. Good catch!